docs/cristin.ttl

```diff
@@ -944,6 +1016,55 @@ It is a self reference.""" ;
         entityDescription.reference.publicationContext.title
     """ .
 
+:tidsskriftnr
+    a               :cristinfield ;
+    rdfs:comment    "Number identifying the specific journal within Cristin" ;
+    rdfs:range      xsd:positiveInteger ;
+    rdfs:domain     :TIDSSKRIFT_Class ;
+    :isignored   "true" .
+
+:nsdkode
+    a               :cristinfield ;
+    rdfs:comment    "A code representing the id in NSD-channelregistry " ;
+    rdfs:range      rdf:nil ; #unknown range
+    rdfs:domain     :TIDSSKRIFT_Class ;
+    :isignored   "true" .
+
+:landkode_utgiver
+    a               :cristinfield ;
+    rdfs:comment    """
+        The code identifying the country where the publisher is located.
+        Example: NO
+        Possibly null.""" ;
+    rdfs:range      xsd:string ;
+    rdfs:domain     :TIDSSKRIFT_Class ;
+    :isignored   "true" .
+
+:status_referee_ordning
+    a               :cristinfield ;
+    rdfs:comment    "Defines whether or not the Journal is peerreviewed (J)=yes, (N)=no, (U)=uncertain and NULL." ;
+    rdfs:range      xsd:string ;
+    rdfs:domain     :TIDSSKRIFT_Class ;
+    :isignored   "true" .
+
+:dato_utgatt
+    a               :cristinfield ;
+    rdfs:comment    """If the journal no longer exists this is the date when it disapeared.
+    Information about what if any: replaced it, it merged into, or what it developed into.
+    Info can be found with NSD and Cristin but is not exported as of today.
+    """ ;
+    rdfs:range      xsd:string ;
+    rdfs:domain     :TIDSSKRIFT_Class ;
+    :isignored   "true" .
+
+:status_serie
+    a               :cristinfield ;
+    rdfs:comment    """Defines wheter it is a bookseries or not (in that case a Journal).
+    (J) declares it to be a bookseries something we dont want/expect to find here""" ;
```

Comment:
Since these appear to be global definitions, "it" and "here" may be difficult to interpret:

"When the value is N, this TIDSSKRIFT object represents a journal. When the value is J, this TIDSSKRIFT object represents a book series. Objects with value J are not expected to be referred to from a TYPE_TIDSSKRIFTPUBLIKASJON object."


Or something like that - "...are only expected to be referred to from a TYPE_BOK_RAPPORT object"


Not sure whether I'm using the correct nomenclature, @axthosarouris

Reply:
Also not super critical, as the field is not likely to be used actively in NVA, and data quality may be sketchy